esmvaltool/preprocessor/_regrid.py

```diff
@@ -140,7 +157,20 @@ def regrid(cube, target_grid, scheme):
         of the form 'MxN', which specifies the extent of the cell, longitude by
         latitude (degrees) for a global, regular target grid.
     scheme : str
-        The regridding scheme to perform, see `regrid.HORIZONTAL_SCHEMES`.
+        The regridding scheme to perform, choose from (arg : func)
+        `linear`: Linear(extrapolation_mode=`mask`),
+        `linear_extrapolate`: Linear(extrapolation_mode=`extrapolate`),
+        `nearest`: Nearest(extrapolation_mode=`mask`),
+        `area_weighted`: AreaWeighted(),
+        `unstructured_nearest`: UnstructuredNearest()
+    lat_offset : bool
+        Offset the grid centers of the latitude coordinate w.r.t. the
+        pole by half a grid step. This argument is ignored if `target_grid`
+        is a cube or file.
+    lon_offset : bool
+        Offset the grid centers of the longitude coordinate w.r.t. the
+        pole by half a grid step. This argument is ignored if `target_grid`
```

Comment:
The offset is not w.r.t. the pole for longitude

Reply:
what is it wrt then? Equator? :grin:

Reply:
Greenwich meridian :gb: